src/components/NavigationTabs/index.tsx

```diff
@@ -71,10 +71,10 @@ const StyledArrowLeft = styled(ArrowLeft)`
 export function SwapPoolTabs({ active }: { active: 'swap' | 'pool' }) {
   return (
     <Tabs style={{ marginBottom: '20px', display: 'none', padding: '1rem 1rem 0 1rem' }}>
-      <StyledNavLink id={`swap-nav-link`} to={'/swap'} isActive={() => active === 'swap'}>
+      <StyledNavLink id={`swap-nav-link`} to={'/swap'} className={({ isActive }) => (isActive ? 'ACTIVE' : undefined)}>
         <Trans>Swap</Trans>
       </StyledNavLink>
-      <StyledNavLink id={`pool-nav-link`} to={'/pool'} isActive={() => active === 'pool'}>
+      <StyledNavLink id={`pool-nav-link`} to={'/pool'} className={({ isActive }) => (isActive ? 'ACTIVE' : undefined)}>
```

Comment:
feel a bit strange. 
shoudn't `StyledNavLink` receive a `isActive: bool` as param and conditionally add the CSS? 

just curious, no need to fix all the Uniswap code. This PR scope is already big

Reply:
<img width="475" alt="image" src="https://user-images.githubusercontent.com/7122625/213409016-56ef9733-418c-4195-9bca-70c0ccc05b12.png">

Internally it uses className.
React router v6 doesn't support `isActive` prop anymore.
https://reactrouter.com/en/main/components/nav-link

Reply:
Anyway, we don't use this file, must be deleted soon as a part of Unswap code.